models.py

```diff
@@ -70,6 +70,7 @@ def __unicode__(self):
 
 MARKER_TYPE_ACCIDENT = 1
 MARKER_TYPE_DISCUSSION = 2
+UNITED_HATZALA_CODE = 2
```

Comment:
Both of them were set to your suggestion... 
Would you like to remove any of them?